.travis.yml

```diff
@@ -1,29 +1,38 @@
+conditions: v1
+
+dist: xenial
 language: python
-sudo: false
-services:
-  - docker
 python:
   - "3.5"
   - "3.6"
+  - "3.7"
 install:
-  - pip install codecov
-  - pip install -r requirements.txt
+  - pip install -U tox
 script:
-  - pytest tests --es_tag 6.0.0
-  - PYTHONASYNCIODEBUG=x pytest tests --es_tag 6.0.0
+  - tox -- --es_tag=${ES_TAG}
 after_success:
+  - pip install codecov
```

Comment:
It'd be nice to put this whole thing into a separate toxenv and invoke it via `tox -e coverage`

Reply:
agree, added